types/awesome-notifications/awesome-notifications-tests.ts

```diff
@@ -1,4 +1,12 @@
-import AWN from 'awesome-notifications';
+import AWN from "awesome-notifications";
```

Comment:
```suggestion
import AWN from 'awesome-notifications';
```
DT repo uses prettier global settings, with single quotes